FRC1076-2016-Competition/test/org/usfirst/frc/team1076/test/mock/MockRobot.java

```diff
@@ -41,7 +52,7 @@ public void setLidarSpeed(double speed) {
 	}
 
 	public SensorData getSensorData() {
-		// TODO Auto-generated method stub
+		// Should this return something else?

```

Comment:
Would it be reasonable to return a mock SensorData object? I remember there was a reason not to do this at first, but I don't remember why.